Add initial "Under construction" page for Eclipse Codewind

sghung opened the pull request:
Adding initial under construction landing pages for Eclipse Codewind.

elsony commented on index.html:
```diff
@@ -0,0 +1,67 @@
+			<h2>Licenses: </h2>
+			<a href="https://projects.eclipse.org/license/epl-2.0" target="_blank">Eclipse Public License 2.0</a>
+			<h2>Legal Issues: </h2>
+			The initial contribution that will form the basis for Codewind will come from the IBM Microclimate project (http://microclimate.dev). The Eclipse platform and VisualStudio Code plugins are already open source and licensed under EPL 2. The common library that these depend on is not open source yet, but it has been legally cleared recently and will be fully donated by IBM under EPL 2. We do not  ...
```
Change "VisualStudio Code" to "Visual Studio Code"

elsony commented on index.html:
```diff
@@ -0,0 +1,67 @@
+				<li>Incremental update of applications running in Docker or Kubernetes</li>
+				<li>Debug and other support for these deployed projects</li>
+				<li>Application performance testing tools</li>
+				<li>Codewind will also provide Eclipse platform and Eclipse Che plugins, and a VisualStudio extension (and possibly others in the future) to allow users of these popular IDEs to create and deploy cloud-native applications in a consistent way from directly within their IDE.</li>
```
Change "VisualStudio" to "Visual Studio Code"

sghung commented:
Updated PR to fix the use of "VisualStudio Code" to "Visual Studio Code" as pointed out by Elson. In addition, fixed a grammar issue on the first sentence.
